lib/emojiKeywords.json

```
@@ -63,5 +63,6 @@
   "photoshop": "photoshop",
   "storm": "thunder_cloud_and_rain",
   "rain": "rain_cloud",
-  "linux": "linux"
+  "linux": "linux",
+  "hacktober": "hacktoberfest"
```

```suggestion
  "hacktober": "hacktoberfest",
  "hacktoberfest": "hacktoberfest"
```
Otherwise this will only work for the word `hacktober`, & would not match `hacktoberfest` or the emoji form `:hacktoberfest:`.